rs/tests/driver/src/driver/prometheus_vm.rs

```diff
@@ -143,6 +144,85 @@ impl PrometheusVm {
         self
     }
 
+    fn sync_k8s_repo_dashboards(logger: Logger) -> Result<()> {
```

Comment:
can you explain what this does and why? I'm a bit confused by why the dashboards need to be "synced"

Reply:
What we expect is to be pointed to the root directory containing the subdirectories used for dashboards. That is what grafana expects. That folder should have the structure like [this](https://github.com/dfinity-ops/k8s/tree/main/bases/apps/ic-dashboards). What that function does is discoveres all the dashboards (which are in subfolders and are `*.json` files) and copies them to a different folder in order to make the structure look the same as it does in [grafana](https://grafana.mainnet.dfinity.network/dashboards). The issue is that, since its k8s and we have some kustomize magic, the links would be a bit broken so I also added the logic that tries to parse the `kustomize.yaml` which has the name of directory that should contain the dashboards in grafana (Basically parse [this file](https://github.com/dfinity-ops/k8s/blob/main/bases/apps/ic-dashboards/application-canisters/kustomization.yaml) for [this line](https://github.com/dfinity-ops/k8s/blob/main/bases/apps/ic-dashboards/application-canisters/kustomization.yaml#L15)). If that function errors in any way, the warning log would be printed and no dashboards would be visible, but the testnet would still be spawned.